.github/workflows/analysis_codeql.yml

```diff
@@ -31,8 +31,6 @@ jobs:
           # The ubuntu-latest label currently points to ubuntu-20.04.
           # Available: ubuntu-22.04, ubuntu-20.04
           os: [ubuntu-latest]
-          # Older versions of GCC are not available via unaltered aptitude repo lists.
-          gcc: ['10']
```

Comment:
By removing this it will use the default compiler, which is most likely newer.
I intentionally set the compiler to a lower version, because they were tons faster during our tests back then.

Reply:
Checking the actions for this PR, the difference is <10% https://github.com/rathena/rathena/actions/runs/5252590012